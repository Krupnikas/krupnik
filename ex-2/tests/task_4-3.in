_________________
weiortbc4urg34iucbg34iugt
14
