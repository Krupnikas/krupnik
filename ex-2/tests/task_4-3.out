weiortbc4urg3_________________4iucbg34iugt
